### DISCUSSION TITLE
How to run single targets sequentially in overall parallel target pipeline?

### DISCUSSION THREAD

**Discussion by MxNl** [ropensci/targets#350] [Help]
Hello,

I have a list of targets that I want to run with

```
plan(multisession)

targets::tar_make_future(workers = future::availableCores() - 1)
```

Unfortunately there is one target in my list that I need to run in sequential mode as it involves a GRASS GIS connection and I couldn't get it work with `plan(multisession)` yet. Is there a way to tell targets to only run this single target not in parallel? Or do I need to break my list in 3 parts (parallel - sequential - parallel)?

I couldn't find any information on that so far.
Thanks a lot for any hint.

**Comment by wlandau** [comment]
As long as you are using transient workers, you can run that one target locally: `tar_target(..., deployment = "main")`. The others should run in parallel with regular orchestration.

**Reply by wlandau** [reply]
Might be worth trying to reproduce that GRASS GIS issue directly inside a `future()` (without `targets`)
